README.md

```diff
@@ -1 +1,34 @@
-# mobile-menu-training
\ No newline at end of file
+
+
+# Mobile-Menu Training ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+This repo host an initial code for the tasks performed during the Code Reviewers training.
+
+Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+## Built With
+- HTML, CSS, JS
+- Linters, Node.js, Git
+
+To get a local copy up and running follow these simple example steps.
+- On your terminal and run the following command to clone the repository
+  
+  `git clone https://github.com/mateo951/mobile-menu-training`
```

Comment:
- [ ] OPTIONAL
I am sorry the previous reviewer did not notice this, and therefore, I will mark it as optional. I would recommend giving all the necessary steps to run the project locally so that any user will be able to run your project locally 👌

After clone 
```
$ cd your_repo

$ npm install

$ npx hint . 
to test for html linting check

$ npx stylelint "**/*.{css,scss}" 
to test for css linting check

```